assembly/src/main/resources/html/cucumber/index.html

```diff
@@ -49,6 +49,7 @@ <h1 class="page-header">Kapua Cucumber Reports</h1>
                     tested in isolation, with all other services mocked and not included in test.

```

Comment:
Hi!
I fixed the headers, some line endings and some misc build problems that popped up because of some develop branch method renames.
Please take a look whenever you have some time.
Thanks!